### ISSUE TITLE
DuckDuckGo search input not rendered properly

### ISSUE THREAD

**Issue by bheesham** [servo/servo#12097]
Build: nightly - 1 July 2016
Website: https://duckduckgo.com

Chrome:
<img src="https://i.imgur.com/6QmYSms.png" alt="Text entry rendered in Chrome">

Servo:
<img src="http://i.imgur.com/OGZTNGw.png" alt="Text entry rendered in Servo">

**Comment by ConnorGBrewster** [comment]
Do note that we use the html version of the duckduckgo.com homepage as https://duckduckgo.com currently has some issues. Try comparing with https://duckduckgo.com/html in Chrome

**Comment by bheesham** [comment]
Good catch. I've updated the issue with the appropriate screenshot for Chrome.

**Comment by atouchet** [comment]
This is an issue with https://duckduckgo.com/html as well.

**Comment by atouchet** [comment]
On the main DuckDuckGo page the magnifying glass icon is off-centre along with the search text. This issue doesn't happen with the HTML version of the page.

![DuckDuckGo icon](https://i.imgur.com/M3LmndW.png)

**Comment by nox** [comment]
This is still true.

**Comment by lumiscosity** [comment]
Text is no longer off-center, but the magnifying glass is. Tested on Arch Linux x86_64, Servo built against commit 633d746e2f1f0bb397e884381bbb7fab89284006, profiling build, compared against Firefox 141.0, Epiphany 48.5 and Chromium 138.0.7204.168 (left to right).

<img width="2328" height="680" alt="Image" src="https://github.com/user-attachments/assets/7ef098ae-f34a-4708-a350-05b1123ae31c" />

HTML version is fine, though:

<img width="2328" height="680" alt="Image" src="https://github.com/user-attachments/assets/a40a2875-273e-4fa7-81c8-75d03927095d" />

**Comment by jdm** [comment]
The magnifying glass is now aligned correctly for me on https://duckduckgo.com/.

Servo:
<img width="500" height="65" alt="Image" src="https://github.com/user-attachments/assets/474f6e8f-c6df-4d38-bb2e-76d103ab92d5" />

Firefox:

<img width="701" height="72" alt="Image" src="https://github.com/user-attachments/assets/0da63732-077c-4468-8e1c-4c58d5b5b9c1" />

**Comment by lumiscosity** [comment]
This is an issue about the old version of the site, but I'm fine with keeping it closed here because after a bit of testing I'm 99% sure it's the same (or at least heavily related) as #17512.
